enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -431,6 +431,17 @@ const theme = {
         backgroundColor: "#95A5A6",
       },
     },
+    
+    collectionBrowser: {
+       breadcrumbs: {
+         expandButton: {
+           backgroundColor: "#767D7C";
+           hoverBackgroundColor: "#69264B";
+           hoverTextColor: "#CE8C8C";
+           textColor: "#8118F4";
```

Comment:
Well the order doesn't make sense

there are 2 main types
1. non-hover with background/text variants
2. hover with background/text variants
or 
1. background with hover/non-hover variants
2. text with hover/non-hover variants

in either case I expect it to follow those orders e.g.
1. backgroundColor
2. hoverBackgroundColor
3. textColor
4. hoverTextColor

or
1. backgroundColor
1. textColor
1. hoverBackgroundColor
1. hoverTextColor

I think you should get the point. It's the the exact order I'm looking for it's the pattern that should be consistent.

Reply:
Agree with this - should be consistent across the entire SDK documentation and the MetabaseComponent type as well.

Reply:
makes sense, I'll change it. I think I wanted to keep it alphabetical for some inexplicable reason

Reply:
@WiNloSt @heypoom let me know if that looks good for you (you can take a look at 1c8843c)

Reply:
Looks good to me.